.github/workflows/updates-testing-nightly.yml

```diff
@@ -0,0 +1,19 @@
+name: fedora-updates-testing
+on:
+  schedule:
+    - cron: '0 1 * * *'
+  # can be run manually on https://github.com/cockpit-project/cockpit-podman/actions
+  workflow_dispatch:
+jobs:
+  fedora-updates-testing:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+
+      - name: Trigger updates-testing scenario
+        run: |
+          make tools/make-bots
+          mkdir -p ~/.config/cockpit-dev
+          echo ${{ github.token }} >> ~/.config/cockpit-dev/github-token
+          ./bots/tests-trigger "-" fedora-38/updates-testing
```

Comment:
@martinpitt is there s a nice trick so that we cannot hardcode `fedora-38`. Maybe make it use `DEFAULT_OS`.

Reply:
`PYTHONPATH=bots python3 -c 'from lib.constants import TEST_OS_DEFAULT; print(TEST_OS_DEFAULT)'`